base/Dockerfile

```diff
@@ -1,22 +1,31 @@
-FROM ubuntu:xenial
+FROM python:2
 
 EXPOSE 5000
 
 RUN useradd --create-home redash
 
 # Ubuntu packages
-RUN apt-get update && apt-get install -y curl && curl https://deb.nodesource.com/setup_6.x | bash - && \
-  apt-get install -y python-pip python-dev build-essential pwgen libffi-dev sudo git-core wget \
-  nodejs \
-  # Postgres client
-  libpq-dev \
-  # for SAML
-  xmlsec1 \
-  # Additional packages required for data sources:
-  libssl-dev libmysqlclient-dev freetds-dev libsasl2-dev && \
+RUN apt-get update && \
+  apt-get install -y curl && \
+  curl https://deb.nodesource.com/setup_6.x | bash - && \
```

Comment:
We can also remove installation of Node in this image and switch to multi FROM form of Dockerfile in `getredash/redash` (I already "implemented" this but didn't push).